LayoutTests/ipc/WebPageProxy_CorrectionPanel.html

```diff
@@ -0,0 +1,4 @@
+<script>
```

Comment:
This is probably not right, other IPC tests have:
<!DOCTYPE html><!-- webkit-test-runner [ IPCTestingAPIEnabled=true ] -->

This might explain the issue @rniwa spotted.

Reply:
If the test case is possible to be added, it could be named something like webpageproxy-correctionpanel-no-crash.html. See the other files in the directory for examples how to write the test.

Reply:
Also:
Originally the value was size of int32_t and the problem was that valid values were 0-3 but 655 was sent.

Now your code changes the value to size of uint8_t. 
So now the test just tests that the IPC wire format checks that the message is correct sized. The test will fail because int32_t is sent instead of uint8_t.

This means that the test case will not check the original underlying bug, e.g. sending values outside the allowed enum range.

So what you could do is to send couple of messages with different values > 3, but with uint8_t type. So this way you could ensure that the original bug is tested, e.g. that the function only allows the values in the enum.

Also, you could then add the case for the disallowed enum value, e.g.  `AlternativeTextTypeDictationAlternatives`.  This is with the caveat that it's not entirely clear to me if `MESSAGE_CHECK`ed values can be tested, or will that too crash.